tf-training/tf-job-operator/base/application.yaml

```diff
@@ -0,0 +1,33 @@
+apiVersion: app.k8s.io/v1beta1
+kind: Application
+metadata:
+  name: "application"
+spec:
+  type: "tf-job-operator"
+  componentKinds:
+    - group: core
+      kind: Service
+    - group: apps
+      kind: Deployment
+    - group: core
+      kind: ConfigMap
+    - group: kubeflow.org
+      kind: TFJob
```

Comment:
should we also include other resource, such as clusterrole, clusterrolebinding, crd and so on?

Reply:
Good question, I was wondering this as well. I did not see clusterRole/bindings/serviceAccounts added in https://github.com/kubeflow/manifests/pull/105.

Reply:
I think for crd, maybe we will need another application.yaml based on current orginazation